src/components/DrawerMenu.vue

```diff
@@ -69,30 +21,40 @@ export default defineComponent({
     <q-item
       v-for="item in menuTop"
       :key="item.name"
-      class="flex items-center gap-8"
+      :to="item.url"
+      class="flex items-center"
       clickable
     >
-      <q-icon :name="item.icon" size="xs"></q-icon>
+      <!-- Link icon -->
+      <q-icon :name="item.icon" size="xs" color="blue-grey-4" class="q-mr-sm" />
+      <!-- Link text -->
       {{ $t(`drawerMenu.${item.name}`) }}
     </q-item>
 
-    <hr />
+    <q-separator color="blue-grey-2 q-my-sm" />
 
     <!-- Menu: Bottom items -->
     <q-item
       v-for="item in menuBottom"
       :key="item.name"
-      class="flex items-center gap-8"
+      :to="item.url"
+      class="flex items-center"
       clickable
     >
```

Comment:
Instead of overriding `q-item` component CSS class props for active menu item use `active-class` [props](https://quasar.dev/vue-components/list-and-list-items/#qitem-api) `active-class="menu-active-item"`

SCSS style:

```scss
<style scoped lang="scss">
.menu-active-item {
  color: $blue-grey-10;
}
</style>
```

Reply:
Agreed. However, there seems to be no purely native way to override the icon color (different from text color) on hover - I still need the .menu-active-item .q-icon SCSS rule.